packages/web-components/src/components/va-language-toggle/va-language-toggle.tsx

```diff
@@ -123,14 +123,29 @@ export class VaLanguageToggle {
     const { language, urls } = this;
     return (
       <Host>
-        <div>
+        <div role="group" aria-label="Language selection">
         {urls.map(({href, lang, label}, i) => {
           const anchorClass = classNames({
             'is-current-lang': lang === language
           });
           const divClass = classNames({
             'inner-div': (i < urls.length - 1)
           })
+          if (lang === language) {
+          return (
+            <div class={divClass}>
+              <a
+                class={anchorClass}
+                href={href}
+                hrefLang={lang}
+                onClick={(e) => this.handleToggle(e, lang, label)}
+                aria-current={"language"}
```

Comment:
`aria-current={"language"}` resolves to aria-current="true", but I figure the extra information can't hurt.